src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/RequestPayloadValueResolver.php

```diff
@@ -52,6 +56,7 @@ final class RequestPayloadValueResolver implements ValueResolverInterface
     public function __construct(
         private readonly SerializerInterface&DenormalizerInterface $serializer,
         private readonly ?ValidatorInterface $validator,
+        private readonly ?TranslatorInterface $translator,
```

Comment:
let's make this default to null and remove the extra args added in tests

Reply:
Same for the validator too ?

Reply:
Let's do it yes